examples/pytest/Readme.md

```diff
@@ -0,0 +1,26 @@
+# How to run this examples
+
+1. Clone the repository
+
+```bash
+git clone https://github.com/qase-tms/qase-python.git
+```
```

Comment:
indent the code to make it a part of the item list, otherwise it'll break the list
```suggestion
	```bash
	git clone https://github.com/qase-tms/qase-python.git
	```
```

Reply:
Fixed